content/docs/capabilities/hosted-authenticate-service.md

```diff
@@ -0,0 +1,74 @@
+---
+id: hosted-authenticate-service
+title: Hosted Authenticate Service
+sidebar_label: Hosted Authenticate Service
+keywords: [hosted authenticate service url, hosted identity provider]
+description: Use Pomerium's Hosted Authenticate Service to set up and deploy Pomerium quickly.
+---
+
+# Hosted Authenticate Service
+
+Pomerium v0.23.0 provides a **Hosted Authenticate Service**, which includes a **Hosted Authenticate Service URL** and a **Hosted Identity Provider** you can use in your Pomerium configuration as an alternative to self-hosting these services.
+
+The Hosted Authenticate Service is available for both open-source Pomerium Core and Enterprise users.
+
+## Why use Hosted Authenticate Service
+
+The Hosted Authenticate Service offers a quicker way for users to deploy and test Pomerium.
+
+### Zero configuration
+
+Including the hosted authenticate URL in your configuration file authenticates your users against our hosted identity provider solution (we use Cognito).
```

Comment:
We will default to the hosted authenticate URL, so, something like:

By default Pomerium will use the hosted identity provider if no `authenticate_service_url` is defined.